test/dynamo/test_export.py

```diff
@@ -32,6 +33,15 @@
     ShapeEnv,
 )
 from torch.testing._internal import common_utils
+from torch.testing._internal.common_cuda import TEST_CUDA
+
+try:
+    import torchvision.models
```

Comment:
@tringwald sorry I was aware of this when reviewed it for the first time. Do you mind creating a repro without depending on torchvision? I think it's better to have tests always enabled so we can make sure something is regressed on all diffs.

Reply:
Sure, I'll try to find a repro without using torchvision. The CUDA part seems necessary though, I wasn't able to reproduce it on CPU.